### PR TITLE
Add FileClaim model, service, controller + refactor

### PR DESCRIPTION
**Pull request by ahsu1230**

### CODE REVIEW

File: winterfell/controllers/authController.js [hunk 1]
```diff
@@ -41,17 +42,17 @@ module.exports = function (app) {
...
           onError: function(errorCode, errorMsg) {
             console.log('[authSignUp] ' + errorCode + ' Error creating user: ' + errorMsg);
-            res.sendStatus(errorCode, errorMsg);
+            res.status(errorCode).send({error: errorMsg});
           },
           onSuccess: function(user) {
             console.log('[authSignUp] Successfully created user ' + user.externalId);
             var extUserId = user.externalId;
-            res.status(200).send({userId: extUserId, redirect: '/'});
+            res.status(http.CODE_SUCCESS).send({userId: extUserId, redirect: '/'});
```
Comment (qwwqwwq) [inline winterfell/controllers/authController.js]: I think its a little confusing to have nonstandard names for the HTTP status codes. Fine with having constants for them, but if we do I think the constant for 200 should be name "OK" etc.

File: winterfell/utils/httpResponses.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+function httpResponses(app) {
+    this.app = app;
+};
+
+module.exports = httpResponses;
+
+// Http Codes (wrapper around standard http response status codes)
```
Comment (qwwqwwq) [inline winterfell/utils/httpResponses.js]: Can we just use https://www.npmjs.com/package/http-status-codes instead of rolling this ourselves?

Comment (ahsu1230) [inline winterfell/utils/httpResponses.js]: Ah ok, I would like to have standardized error message constants so I'll keep that for now

**Comment by ahsu1230** [comment]
I'll keep these changes smaller in the future.
Sorry, just wanted to refactor to make things neater.

**Pull request merged**